src/main/java/de/tum/in/www1/artemis/versioning/VersionRange.java

```diff
@@ -0,0 +1,22 @@
+package de.tum.in.www1.artemis.versioning;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Depicts a range of versions and can contain one or two values.
+ * The first value is the lower bound and the second, optional, value is the upper bound.
+ */
+@Target({ ElementType.METHOD })
+@Retention(RetentionPolicy.RUNTIME)
+public @interface VersionRange {
+
+    /**
+     * Specifies the range by version numbers
+     *
+     * @return array of version numbers
+     */
+    int[] value();
```

Comment:
Why did you use an array instead of two separate values?

Reply:
To avoid always having the `abcd = ` prefix.

I could also set value as an alias for `start` and only if there is an end version, you'd have to specify it. Would you prefer this?

Reply:
With specific values it would be a bit clearer and easier to understand what values can be used for the annotation. But I don't have hard feelings against the array

Reply:
I changed it